cmd/tls.go

```diff
@@ -18,17 +18,17 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// NewInstall creates the `install` command
-func NewInstallCmd() *cobra.Command {
+// NewTLSCmd return the tls command.
+func NewTLSCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:                   "install",
+		Use:                   "tls",
 		TraverseChildren:      true,
 		DisableFlagsInUseLine: true,
-		Short:                 "Install a component with falcoctl",
-		Long:                  `Install a component with falcoctl`,
+		Short:                 "Handle certificates with falcoctl",
```

Comment:
Is this a right choice for the description? Can we find better?

Reply:
My proposal is: `Generate and install TLS material for Falco`